charts/backstage/values.yaml

```diff
@@ -154,6 +149,36 @@ backstage:
   #     cpu: 100m -->
   resources: {}
 
+  # -- Readiness Probe
+  # <br /> Ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes
```

Comment:
Please explain here that simply enabling the probe doesn't make is work. There's no out-of-the-box endpoint available in the instance unless explicitly configured.

See the comments in the issue: https://github.com/backstage/charts/issues/95#issuecomment-1541005425